src/mocks/FileUploadMocks.js

```diff
@@ -0,0 +1,17 @@
+import {rest} from 'msw';
+
+export const BASE_URL = 'http://localhost:8000/api/v2/';
+export const ENDPOINT = 'formio/fileupload/';
+
+export const mockFileUploadPost = rest.post(`${BASE_URL}${ENDPOINT}`, (req, res, ctx) => {
+  const body = {
+    url: `${BASE_URL}${ENDPOINT}/test`,
+    name: 'test',
+    size: 'test',
+  };
+  return res(ctx.status(200), ctx.json(body));
+});
+
+export const mockFileUploadError = rest.post(`${BASE_URL}${ENDPOINT}`, (req, res, ctx) => {
+  return res(ctx.status(400));
```

Comment:
please add an example error in the response body - I remember this requires some cruft so you can trigger an invalid upload on a real form and then look at the response data for an example via the browser debugger.